source/features/follow-file-renames.tsx

```diff
@@ -7,48 +7,69 @@ import features from '.';
 import * as api from '../github-helpers/api';
 import GitHubURL from '../github-helpers/github-url';
 
-interface File {
-	previous_filename: string;
-	filename: string;
+export interface File {
+	previous_name: string | undefined;
+	name: string;
 	status: string;
+	url: string;
 }
-// eslint-disable-next-line import/prefer-default-export
-export async function getCommitInfo(oid: string): Promise<AnyObject> {
+export interface CommitInfo {
+	file: File | undefined;
+	parentSha: string | undefined;
+	date: Date;
+	url: string;
+}
+
+export async function getChangesToFileInCommit(sha: string, filePath: string): Promise<CommitInfo> {
 	// API v4 doesn't support it: https://github.community/t/what-is-the-corresponding-object-in-graphql-api-v4-for-patch-which-is-available-in-rest-api-v3/13590
-	return api.v3(`commits/${oid}`);
+	const commitObject = await api.v3(`commits/${sha}`);
+	const commitInfo: CommitInfo = {
+		file: undefined,
+		parentSha: commitObject.parents[0],
+		date: commitObject.commit.committer.date,
+		url: commitObject.html_url,
+	};
+	for (const f of commitObject.files) {
+		if ([f.filename, f.previous_filename].includes(filePath)) {
+			commitInfo.file = {
```

Comment:
You can create the object and return all the information at once
```ts
			return {
				commit: {/*etc*/},
				file: {/*etc*/}
			}
```

The interface can be called `FileChanges` instead of CommitInfo and File